agent/repowise.py

```
@@ -0,0 +1,233 @@
+        entry_patterns = ["main.py", "app.py", "server.py", "proxy.py", "index.js", "index.ts", "run.sh", "Makefile"]
+        found = []
+        for pattern in entry_patterns:
+            matches = list(self.root.glob(pattern)) + list(self.root.glob(f"*/{pattern}"))
+            for m in matches:
+                found.append(str(m.relative_to(self.root)))
+        return list(set(found))
+
+    def get_git_health(self) -> Dict[str, Any]:
+        """Basic git health metrics."""
+        try:
+            commit_count = subprocess.run(["git", "rev-list", "--count", "HEAD"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)